### PR TITLE
Added GET endpoint for newStoryModel and newStoryRouter for new stories schema

### PR DESCRIPTION
**Pull request by allraec**
# Description

Fixes #

- What work was done?
 Added a new Stories/Story model and router with GET endpoint for a story that corresponds to the new Stories schema in place.
- Why was this work done?
 This was done to support the new Stories schema with new endpoints.
- What feature / user story is it for?
 New Story data schema.
- Any relevant links or images / screenshots
 n/a

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [x] Complete, but not tested (may need new tests)

## Has This Been Tested

- [x] No

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

### CODE REVIEW

**Review by SethMacP — COMMENTED** [review]
I would encourage writing tests to ensure functionality is working and stays working with further changes down the road. They aren't fun but they are useful.

I left a comment on one of the commits about posting a status code with your error message, but I believe that is taken care of by using the 'crudOperationsManager'. If it isn't I would encourage adding it.

File: api/stories/newStoryRouter.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const router = require('express').Router();
+const { crudOperationsManager } = require('../../lib');
+const Stories = require('./newStoryModel');
+
```
Comment (ChadDiaz) [inline api/stories/newStoryRouter.js:4]: code looks great.  I feel that you should comment your code.  Looking at the old 'storyModel', each function has a very descriptive explanation of what it is doing:  
`/**
 * Queries the database for a specific story with given ID
 * @param {number} ID the ID to search for in the database
 * @returns {Promise} a promise that resolves to a story object
 */`
Putting in good comments will ensure that not only you, but others that come after you, will know exactly what you were wanting to accomplish.

**Review by ChadDiaz — CHANGES_REQUESTED** [review]
Looks great. Put some good comments about what your code is doing.  Look at the old storyModel for an example.

File: api/stories/newStoryRouter.js (hunk 1 above)
Comment (allraec) [reply api/stories/newStoryRouter.js:4]: Thanks! I applied comments for the newStoryModel.

**Review by ChadDiaz — APPROVED** [review]
Awesome!

File: api/stories/newStoryRouter.js (hunk 1 above)
Comment (jproctor-rebecca) [reply api/stories/newStoryRouter.js:4]: I second the comments request.  I think of comments as a common or bridge language in cross-functional and longitudinal teams.  They can help members with a variety of skillsets working concurrently or consecutively get on the same page quickly.

**Review by jproctor-rebecca — APPROVED** [review]

**Comment by jproctor-rebecca** [comment]
Looks great!  I left a comment about commenting your code.

**Review by Antashma — APPROVED** [review]
Looks good!  I was going to ask about testing and I just saw your comment in Slack about adding testing in the next PR.  Good to know!

**Review by kdneldor — APPROVED** [review]
Looks great Allison! Looking forward to seeing those tests in the next PR.

File: api/stories/newStoryModel.js [hunk 2]
```diff
@@ -1,10 +1,5 @@
 const db = require('../../data/db-config');
 
-/**
```
Comment (tinomen) [inline api/stories/newStoryModel.js]: Would have been better to leave it in place but update the content

**Pull request merged**